python/amici/sbml_import.py

```diff
@@ -150,43 +152,48 @@ def loadSBMLFile(self, SBMLFile):
         self.sbml = self.sbml_doc.getModel()
 
     def sbml2amici(self,
-                   modelName,
-                   output_dir=None,
-                   observables=None,
-                   constantParameters=None,
-                   sigmas=None,
-                   verbose=False,
-                   assume_pow_positivity=False,
-                   compiler=None,
-                   allow_reinit_fixpar_initcond = True
+                   modelName: str,
```

Comment:
Nice! What is doxygen doing with this type information? Can we remove @type instructions in the docstring? Would prefer to encode type info in code and not in docstrings that are prone to be stalled.

Reply:
Yeah, I found typehinting extremely useful. A proper IDE will tell you that you are passing the wrong arguments. Didn't check what doxygen makes of it yet, would be way more readable and up to date, yes.

Reply:
Doesn't work with doxygen :(. Maybe this can be changed somehow.

Reply:
https://github.com/UwePfautsch/doxyfilter_python should work with it, but there were other issues. Will need some further checking...

Reply:
Doxypy cannot identify arguments with typehints. We might want to fix the signature parsing ourselves.  

> /home/dweindl/src/AMICI-devel/python/amici/sbml_import.py:241: warning: argument 'observables' of command @param is not found in the argument list of SbmlImporter::sbml2amici(self, modelName)